### ISSUE TITLE
[BUG] - Typeerror: tabulator is not a constructor

### ISSUE THREAD

**Issue by leokyohan** [angeliski/vue-tabulator#83]
**Describe the bug**
When I'm trying to use VueTabulator with Nuxt.js, I'm getting the following error
Typeerror: tabulator is not a constructor

**VueTabulator Info**
- Which version of VueTabulator are you using? : Both Tabulator and VueTabulator is Latest Version. I just installed it now.
"vue-tabulator": "^1.2.2",
"tabulator-tables": "^4.8.1",

I created vuetabulator.js file in plugins in nuxt with below content

import Vue from 'vue';
import VueTabulator from 'vue-tabulator';
import 'tabulator-tables/dist/css/tabulator_simple.min.css'
Vue.use(VueTabulator)

Then in   nuxt.config.js file I did
plugins: [
  { src: '~/plugins/vuetabulator.js', mode: 'client', ssr: false },
],

Then in my vue file I'm calling the tabulator like this
<div slot="no-body">
 <VueTabulator
                v-model="userData"
                :options="TableOptions"                
              />
</div>

And my Table options are as below in data section

TableOptions: {
              height: "1650",
              layout: "fitDataStretch",
              pagination: "local",
              paginationSize: 50,
              selectable: 1,
              movableColumns: true,
              autoResize: false,        
              columns: [
                {
                  title: "ID",
                  formatter: "rownum",
                  hozAlign: "center",
                  frozen: true
                },
                {
                  title: "Full Name",
                  field: "Full_Name",
                  sorter: "string",
                  frozen: true,
                  headerFilter: "select",
                  headerFilterParams: { values: true }
                }, 
     ]}
            
This was working fine in my Vue project. But when I migrated this to my nuxt project, I'm getting this error.
What might be going wrong?

**Comment by angeliski** [comment]
Hey @leokyohan 
I will try to see what is happening, in the meantime, can you provide a repository with the minimum to reproduce that issue? 
I didn't work using Nuxt.js and this could help me a lot

**Comment by leokyohan** [comment]
Hi @angeliski 

Thanks for the reply.

I have created the following repository with Nuxt and VueTabulator.
https://github.com/leokyohan/Nuxt_Tabulator.git 

You can just clone and give a npm run dev.
You will get the error in the console.

**Comment by leokyohan** [comment]
Any updates on this?

**Comment by leosalass** [comment]
> Any updates on this?

Hi did you solve the problem? I am having the same issue with vue-cli

**Comment by leosalass** [comment]
> Any updates on this?

changing the version solved our problem. (vue-cli)
npm install tabulator-tables@4.7.2 --save

**Comment by leokyohan** [comment]
> > Any updates on this?
> 
> changing the version solved our problem. (vue-cli)
> npm install tabulator-tables@4.7.2 --save

Awesome...This solution worked for me also... @leosalass Thanks a lot.

@angeliski any idea what is the issue when we use the latest tabulator? The version I used was "tabulator-tables": "^4.8.1"

**Comment by exetico** [comment]
@leokyohan - Check out https://github.com/olifolkerd/tabulator/issues/2977

Hopefully @angeliski are able to fix that part.

It must be related to the way Tabulator is loaded.
https://github.com/angeliski/vue-tabulator/blob/e02ac48bc107292a7379bc56f4be56ccf7cd350e/src/components/TabulatorComponent.vue#L20

Vs. 
```js
import Tabulator from 'tabulator-tables';
```

I've tried to change it, and builded the plugin - and it's working fine with `tabulator-tables@4.8.1`. That's great.

But it's reporting an error during the build-progress. So I'm not sure if this is the right fix, or if some other things needs to be changed/done in another way. Therefore, i won't create a pull request.
```
ERROR in .../vue-tabulator/src/components/TabulatorComponent.vue
16:23 File '.../vue-tabulator/node_modules/@types/tabulator-tables/index.d.ts' is not a module.
    14 | import mergeWith from 'lodash.mergewith'
    15 | import merge from '../utilities/merge'
  > 16 | import Tabulator from 'tabulator-tables';
       |                       ^
    17 | import eventFactory from '../feature/event-factory'
    18 | import cellEvents from '../feature/events/cell-events'
    19 | import rowEvents from '../feature/events/row-events'
```

TL;DR: I'm not sure how to implement it the right way, as I'm fairly new to NPM - but it can be fixed by using `import` instead of defining the const/variable.

**Comment by angeliski** [comment]
Hey @exetico Your clue is right, but I got the same error.
I open an issue to discuss that in the Tabulator repo, after that we can fix vue-tabulator to works in the last Tabulator release.

**Comment by Jojoshua** [comment]
> Hi @angeliski
> 
> Thanks for the reply.
> 
> I have created the following repository with Nuxt and VueTabulator.
> https://github.com/leokyohan/Nuxt_Tabulator.git
> 
> You can just clone and give a npm run dev.
> You will get the error in the console.

When I cloned and tried to build I get the following error:
`ERROR  Package exports for '...\Nuxt_Tabulator\node_modules\colorette' do not define a valid '.' target `

**Comment by angeliski** [comment]
Hey people, 
I release the version 1.3.0 fixed

Thanks, @Jojoshua for the help
